app/dash_app.py

```diff
@@ -278,13 +385,49 @@ def plot_coherence(coh_long):
 temporal_view = Graph(
     id='coherence-matrix',
     figure=plot_coherence(_read_coherence(_coherence_csv(INITIAL_TARGET))),
+    # figure=plot_baseline(_read_baseline(_baseline_csv(INITIAL_TARGET)),
+    #                      _read_coherence(_coherence_csv(INITIAL_TARGET))),
     style={'height': TEMPORAL_HEIGHT},
 )
 
+tab_style = {
+    'borderBottom': '1px solid #d6d6d6',
+    'color': 'black',
+    'padding': '6px',
+    'fontWeight': 'bold',
+    'font-size': '11px',
+}
+
+tab_selected_style = {
+    'align-items': 'top',
+    'borderTop': '1px solid #d6d6d6',
+    'borderBottom': '1px solid #d6d6d6',
+    'backgroundColor': '#119DFF',
+    'color': 'black',
+    'font-size': '11px',
+    'padding': '6px'
+}
+
+baseline_tab = Tabs(id="tabs-example-graph",
+                    value='tab-1-coherence-graph',
+                    children=[Tab(label='Coherence',
+                                  value='tab-1-coherence-graph',
+                                  style=tab_style,
+                                  selected_style=tab_selected_style),
+                              Tab(label='B-Perp',
+                                  value='tab-2-baseline-graph',
+                                  style=tab_style,
+                                  selected_style=tab_selected_style),
+                              ],
+                    style={'width': '10%',
+                           'height': '25px'},
+                    vertical=False)
+
 app.layout = dbc.Container(
     [
         dbc.Row(dbc.Col(selector, width='auto')),
         dbc.Row(dbc.Col(spatial_view), style={'flexGrow': '1'}),
+        dbc.Row(dbc.Col(baseline_tab)),  # add into row below
```

Comment:
Strange, I really thought you were going to make the "temporal_view" tabbed, not add something in a row above it.